docs/source/guides/deployment.rst

```diff
@@ -54,6 +54,8 @@ In case of valid assumptions, the only thing that remains is to launch the daemo
 |               | used by peer to sign the blocks                                 |
 +---------------+-----------------------------------------------------------------+
 
+.. Attention:: You have to specify `--overwrite_ledger` to overwrite existing transactions. If flag is not specified and transactions exist the daemon will not start and exit.
```

Comment:
> existing transactions

Imo should be `existing block storage`

> If flag is not specified and transactions exist the daemon will not start and exit

What about the case of continuing node operation? For example after temporary peer failure

Reply:
That makes sense. In fact, I had similar doubt. But I am not sure what will be the solution in that case. How should I rephrase this sentence ?

Reply:
I think if the flag is not specified it basically tries to reload existing one. And that is ok and do not required to mention (i think there's some notice in docs for that)

Reply:
>* if flag is not set and there are blocks in ledger, we output a message in log_ and exit.
This was mentioned by lebdron and it says to `exit` the daemon. So does it not mean that the daemon will no run ? 

Also, the code [here](https://github.com/hyperledger/iroha/blob/develop/irohad/main/irohad.cpp#L144) ` irohad.dropStorage();`  looks like it **does not** reload, instead drops it.

Reply:
Yes, there is a phrasing issue in my original post. I was referring to `--genesis_block` flag, not `--force` or `--overwrite_ledger`.
https://github.com/hyperledger/iroha/pull/1300#discussion_r187008979
So "if we specify --genesis_block, --overwrite_ledger is not set, and there are block in ledger, we output a message in log_ and exit"

Reply:
Got it ! thanks !